lib/igniter/libs/swoosh.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Igniter.Libs.Swoosh do
+  @moduledoc "Codemods & utilities for working with Swoosh"
+
+  @doc "Lists all mailers found in the project."
```

Comment:
```suggestion
  @doc "Lists all project modules that call `use Swoosh.Mailer`."
```

IMO, best to be very explicit here.